<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clevo xSM wmi Utility</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonApply">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="stateCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Light on</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_mode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Mode</string>
    </property>
    <widget class="QRadioButton" name="radio_flash">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Flash</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">radioModeGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radio_wave">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Wave</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">radioModeGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radio_dance">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Dance</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">radioModeGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radio_tempo">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Tempo</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">radioModeGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radio_cycle">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cycle</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">radioModeGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radio_random_color">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Random Color</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">radioModeGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radio_breathe">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Breathe</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">radioModeGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radio_custom">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">radioModeGroup</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_brightness">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>591</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Brightness</string>
    </property>
    <widget class="QSlider" name="slider_brightness">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>551</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_color">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>591</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Backlight colors (if Custom mode)</string>
    </property>
    <widget class="QLabel" name="labelExtraColor">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>90</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Extra</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRightColor">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Right side</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCenterColor">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>53</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Center</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectRight">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="selectExtra">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>50</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelLeftColor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>53</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Left side</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectCenter">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="selectLeft">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="notifyFailure">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;One or more item was hidden from view.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Please read terminal output for more.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="radioModeGroup"/>
 </buttongroups>
</ui>
